src/server/database.js

```diff
@@ -19,6 +19,28 @@ export const clearDB = () => {
   });
 };
 
+export const putPromise = (key, value) => {
+  return new Promise((resolve, reject) => {
```

Comment:
Right. However there are many lagacy codes that are using levelup put and get method, so  to reduce confusion I added the "Promise" postfix. Later merging current PRs, let's talk about adapting promise in db :)